Doc/source/optional.rst

```diff
@@ -1,142 +1,112 @@
+This module provides an ElementTree-like API for reading and writing XML files. It supports two backend options:
 
-*Extra:* ``lxml``
+- Built-in `xml.etree` module
+- `lxml` preferred for improved speed and security.
 
+*Extra:* `lxml`
 
 :py:mod:`fontTools.ufoLib`
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Package for reading and writing UFO source files; it requires:
+----------------
 
-* `fs <https://pypi.org/pypi/fs>`__: (aka ``pyfilesystem2``) filesystem abstraction layer.
+Package for reading and writing UFO source files. Dependencies include:
 
-* `enum34 <https://pypi.org/pypi/enum34>`__: backport for the built-in ``enum`` module (only required on Python < 3.4).
-
-*Extra:* ``ufo``
+- `fs <https://pypi.org/pypi/fs>`__: (aka ``pyfilesystem2``) filesystem abstraction layer
+- `enum34 <https://pypi.org/pypi/enum34>`__ backport for the built-in ``enum`` module (module for Python versions < 3.4)
 
+*Extra:* `ufo`
 
 :py:mod:`fontTools.ttLib.woff2`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Module to compress/decompress WOFF 2.0 web fonts; it requires:
+---------------------
 
-* `brotli <https://pypi.python.org/pypi/Brotli>`__: Python bindings of the Brotli compression library.
+Module for compressing and decompressing WOFF 2.0 web fonts. Requires:
 
-*Extra:* ``woff``
+- `brotli <https://pypi.python.org/pypi/Brotli>`__: Python bindings for the Brotli compression library.
 
+*Extra:* `woff`
 
 :py:mod:`fontTools.unicode`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-----------------
 
-To display the Unicode character names when dumping the ``cmap`` table
-with ``ttx`` we use the ``unicodedata`` module in the Standard Library.
+Displays Unicode character names when working with the `cmap` table dumps,with `ttx` we use the `unicodedata` module in the Standard Library. 
 The version included in there varies between different Python versions.
-To use the latest available data, you can install:
-
-* `unicodedata2 <https://pypi.python.org/pypi/unicodedata2>`__: ``unicodedata`` backport for Python
-  3.x updated to the latest Unicode version 14.0. Note this is not necessary if you use Python 3.11
-  as the latter already comes with an up-to-date ``unicodedata``.
+To use the latest Unicode version, install:
 
-*Extra:* ``unicode``
+- `unicodedata2 <https://pypi.python.org/pypi/unicodedata2>`__: backport of `unicodedata` for Python 3.x, updated to unicode 14.0. Not necessary with Python 3.11.
 
+*Extra:* `unicode`
 
 :py:mod:`fontTools.varLib.interpolatable`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------
 
-Module for finding wrong contour/component order between different masters.
-It requires one of the following packages in order to solve the so-called
-"minimum weight perfect matching problem in bipartite graphs", or
-the Assignment problem:
+Module for resolving contour or component order between different masters. Requires following packages to solve 'minimum weight perfect matching problem in bipartite graphs':
```

Comment:
Needs a "the" before "following packages" ... or "one of the"?

Reply:
I think that, if the Assignment Problem is well-defined enough to have a name, then it'd be ideal to drop a link about it where the name is mentioned. Wikipedia maybe?